src/main/pileup.js

```diff
@@ -79,13 +79,13 @@ function create(elOrId: string|Element, params: PileupParams): Pileup {
                          initialRange={params.range} />, el);
   return {
     setRange(range: GenomeRange) {
-      if (reactElement == null) {
+      if (reactElement === null) {
```

Comment:
gotcha! I changed it only because jshint was complaining about it, but good to know the dynamics behind this.